app/models/dor_object_workflow_status.rb

```diff
@@ -0,0 +1,18 @@
+class DorObjectWorkflowStatus
+  attr_reader :pid
+
+  ##
+  # @param [String] pid in format "druid:abc123def4567"
+  def initialize(pid)
+    @pid = pid
+  end
```

Comment:
You might to init something to hang onto the `Dor::WorkflowService`, something like:
`@workflow = Dor::WorkflowService`
If it's in one place and it needs to change later, it will be easy.  (Change could be coming.)